app/scripts/modules/appengine/src/serverGroup/configure/wizard/basicSettings.controller.ts

```diff
@@ -104,6 +108,54 @@ class AppengineServerGroupBasicSettingsCtrl implements IController {
     }
   }
 
+  public readonly excludedGcsArtifactTypes = [
```

Comment:
It looks like this predates this PR, but I wonder if it would make more sense if each artifact component just explicitly stated which artifact types it wanted rather than enumerated a list of all the ones to exclude.  This seems reasonably fragile, as any new artifact type will now show up in this stage unless the code here is updated to actively exclude it.

Reply:
Actually it looks like there's an `excludeAllTypesExcept` in `ArtifactTypePatterns` that would eliminate the two hard-coded lists below (though arguably just having an `include-types` in the artifact selector might be cleaner).

Reply:
You are right, Thanks!